Implement  ovh_iploadbalancing_failover_ip_attach resource/data (#437)
ovh/terraform-provider-ovh: website/docs/d/iploadbalancing_failover_ip_attach_html.markdown

---
subcategory : "Load Balancer (IPLB)"
---

> # ovh_iploadbalancing_vrack_network (Data Source)

Reviewer: What is the correct name of the documetation page? :-) 

"ovh_iploadbalancing_failover_ip_attach" ? ;-)

> Use this data source to get list failover ip routed for your IPLoadbalancer associated with your OVHcloud account.

Reviewer: ```suggestion
Use this data source to list all failover IP routed to this IPLB.
```

## Example Usage

```hcl

> data ovh_iploadbalancing_failover_ip_attach failoverip {

Reviewer: ```suggestion
data "ovh_iploadbalancing_failover_ip_attach" "failoverip" {
```
